emsdk

```diff
@@ -174,7 +174,8 @@ def vs_filewhere(installation_path, platform, file):
     return ''
 
 
-CMAKE_GENERATOR = 'Unix Makefiles'
+# Use ninja build tool if present
+CMAKE_GENERATOR = 'Ninja' if which('ninja') is not None else 'Unix Makefiles'
```

Comment:
Might be better to move the logic of selecting Generator to `else` after `if WINDOWS`.
Also might be worth to investigate if generator couldn't be passed to `./emsdk` with a flag i.e. `--generator=` then if given it will be selected unconditionally, otherwise auto-deduced

Reply:
Thank you for the suggestion! I've rebased and included a new commit that hopefully addresses your comments.